SCF-Assignment-2/Question 1/ConversionClass.java

```diff
@@ -0,0 +1,40 @@
+package Hexadecimal;
+
+/*
+ * this class includes function which will convert hexadecimal number to decimal number and vice versa.
+ */
+public class ConversionClass {

```

Comment:
Class name never contains Class suffix. Can name it as HexadecimalConversion.
Also we can make the methods in this class as public static as there is no need to create object for this.